armi/physics/neutronics/macroXSGenerationInterface.py

```diff
@@ -107,6 +125,7 @@ class MacroXSGenerationInterface(interfaces.Interface):
     def __init__(self, r, cs):
         interfaces.Interface.__init__(self, r, cs)
         self.macrosLastBuiltAt = None
+        self.minimumNuclideDensity = cs["minimumNuclideDensity"]
```

Comment:
Consider replacing with it's `CONF_` equivalent
https://github.com/terrapower/armi/blob/f17956333355ea8f19e709e2defb0d7aef94f40b/armi/physics/neutronics/settings.py#L72

Reply:
https://github.com/terrapower/armi/pull/1248/commits/8c3b188a2e4a5144a26205107a165a612fd33e6d